include/openssl/configuration.h.in

```diff
@@ -24,6 +24,7 @@ extern "C" {
 /*
  * OpenSSL was configured with the following options:
  */
+#define OPENSSL_NO_EGD
```

Comment:
I'd move this down to the end next to the `NO_EC2M` one that Watson added.

Reply:
Done.  (And I tidied up the EC2M spacing while I was there.)